test_runner/performance/test_perf_olap.py

```diff
@@ -108,7 +125,28 @@ def run_psql(env: RemoteCompare, labelled_query: LabelledQuery, times: int) -> N
         log.info(f"Run {run}/{times}")
         with env.zenbenchmark.record_duration(f"{label}/{run}"):
             env.pg_bin.run_capture(["psql", connstr, "-c", query], env=environ)
+    if os.getenv('TEST_OLAP_COLLECT_EXPLAIN', 'false').lower() == 'true':
+        log.info(f"Explaining query {label}")
+        run += 1
+        with env.zenbenchmark.record_duration(f"{label}/EXPLAIN"):
+            env.pg_bin.run_capture(["psql", connstr, "-c", EXPLAIN_STRING+query], env=environ)
```

Comment:
```suggestion
            env.pg_bin.run_capture(["psql", connstr, "-c", f"{EXPLAIN_STRING} {query}"], env=environ)
```

And then we can delete an extra space from the end of `EXPLAIN_STRING`

Reply:
ok